.gitlab-ci.yml

```diff
@@ -0,0 +1,107 @@
+image: smk762/qt-ci:latest
+
+stages:
+  - build
+  - upload_zip
+  - upload_portable
+  - test
+  
+variables:
+  GIT_SUBMODULE_STRATEGY: recursive
+  DOCKER_DRIVER: overlay2
+  PIP_CACHE_DIR: "$CI_PROJECT_DIR/.cache/pip"
+  VCPKG_DEFAULT_BINARY_CACHE: "$CI_PROJECT_DIR/.cache/vcpkg"
+  QT_VERSION: "5.15.2"
+
+cache:
+  - key: $CI_COMMIT_REF_SLUG
+    paths:
+      - $CI_PROJECT_DIR/bundled/linux/atomicdex-desktop-linux-${CI_COMMIT_SHORT_SHA}.zip
+      - $CI_PROJECT_DIR/bundled/linux/atomicdex-desktop-${CI_COMMIT_SHORT_SHA}-x86_64.AppImage
+  - key: $CI_PROJECT_NAME
+    paths:
+      - "$CI_PROJECT_DIR/.cache/vcpkg"
+    
+
+linux:build:
+  stage: build
+  before_script:
+    - cd $CI_PROJECT_DIR
+    - mkdir -p .cache/pip
+    - mkdir -p .cache/vcpkg
+
+  script:
+    # Job Variables
+    - echo $CI_COMMIT_REF_SLUG
+    - echo $CI_COMMIT_AUTHOR
+    - echo $CI_COMMIT_BRANCH
+    - echo $CI_COMMIT_SHA
+    - echo $CI_COMMIT_SHORT_SHA
+    - echo $CI_PROJECT_PATH
+    - echo $CI_PROJECT_NAME
+    - echo $QT_VERSION
+
+    # Job Path/Env variables
+    - export DEBIAN_FRONTEND=noninteractive
+    - export QT_INSTALL_CMAKE_PATH=/opt/Qt/$QT_VERSION/gcc_64/lib/cmake
+    - export QT_ROOT=/opt/Qt/$QT_VERSION
+    - export Qt5_DIR=/opt/Qt/$QT_VERSION/gcc_64/lib/cmake/Qt5
+    - export PATH=/opt/Qt/$QT_VERSION/gcc_64/bin:$PATH
+    - export PATH=${PATH}:/cmake-3.19.0-rc3-Linux-x86_64/bin;
```

Comment:
Probably better to use a normal version of CMake, an official one rather a release candidate

Reply:
will go to use cmake-3.20.5-linux-x86_64 now